our_site/planner/views.py

```diff
@@ -173,3 +173,14 @@ def edit_event(request, event_id):
             request
         )
     )
+
+
+def event_home(request, event_id):
```

Comment:
Events are viewable without being logged in and without being a person who has actually been invited to an event. There should be guards in place so that you have to be invited to an event or own it to be able to view it.